src/tools/antool/listsyscalls.py

```diff
@@ -412,7 +412,7 @@ def is_supported(self, syscall):
                 return RESULT_UNSUPPORTED_AT_ALL
 
             # the following syscalls are not supported YET
-            if syscall.name in ("dup", "dup2", "dup3", "flock"):
```

Comment:
we don't have a dup3, YET.

Reply:
LOL. So now we should revert part of PR#338 (which you approved, BTW)... Or are you going to implement dup3 soon?

Reply:
Well, I guess I can implement dup3